### PR TITLE
COOKIE BEAR - prank

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
Prank.

I didn't examine this in detail, but the code seems to make use of some tricks maybe for hiding its identify.

I'm not sure how to use the CHECK FOO file.  Looks like it should set up a translation for HACTRN and then reload it from TS CHECK.

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
Hold on a bit.  I don't see how someone could send a message back to the cookie bear.  Maybe there's some special magic I haven't done.

File: doc/programs.md [hunk 1]
```diff
@@ -39,6 +39,7 @@
 - COMIFY, convert HEX to COM format.
 - COMPLR, Lisp compiler.
 - COMSAT, mail server.
+- COOKIE BEAR, prank.
```
Comment (atsampson) [inline doc/programs.md]: Something like "- CHECK, Cookie Bear prank." might be clearer, although I appreciate clarity may be missing the point...

**Comment by larsbrinkhoff** [comment]
That could be part of the puzzle.  Maybe you should login as BEAR and then :XFILE CHECK FOO.  But then I don't see how to pass the victim user name on the JCL to TS CHECK.

**Comment by larsbrinkhoff** [comment]
Yeah ok, maybe that's just as well.

@atsampson, yes I'd like to preserve a little bit of the mystery.

**Pull request merged**